images/linux/scripts/installers/swift.sh

```diff
@@ -19,11 +19,13 @@ download_with_retries $swift_tar_url "/tmp" "$swift_tar_name"
 # Download swift pgp key
 download_with_retries "https://swift.org/keys/all-keys.asc" "/tmp" "all-keys.asc"
 # Import swift pgp key
-gpg --import /tmp/all-keys.asc
+gpg --no-default-keyring --keyring swift --verify --import /tmp/all-keys.asc
```

Comment:
`--verify` and  `--import` options can't be used together as both represent actions

Reply:
oops... Fixed